src/utils/misc.tsx

```diff
@@ -28,34 +28,44 @@ export function makeLazy<T>(factory: () => T): () => T {
     return () => cache ?? (cache = factory());
 }
 
-type AwaiterRes<T> = [T, any, boolean, () => void];
+type AwaiterRes<T> = [T, any, boolean];
+interface AwaiterOpts<T> {
+    fallbackValue: T,
+    deps?: unknown[],
+    onError?: (e: unknown) => unknown,
```

Comment:
```suggestion
    onError?(e: any): void,
```
unknown is absolute pain to work with

Reply:
this was ur code lol

Reply:
lol really